aws-observability-terraform/app-modules/rds/app.tf

```diff
@@ -386,7 +386,7 @@ module "rds_module" {
     },
     "RdsPostgreSQLHighErrors" = {
       monitor_name         = "Amazon RDS PostgreSQL - High Errors"
-      monitor_description  = "This alert fires when we detect Postgres logs show high rate of error/fatal logs"
+      monitor_description  = "This alert fires when we detect high rate (>10) of error/fatal logs in Postgres logs over a 5 minutes time period"
```

Comment:
rate is not expected here.
Should it be
This alert fires when we detect high number (>10) of error/fatal logs in Postgres logs over a 5 minutes time period